docs/blogs/EnhancedDevPreviewGatewayAPI/GettingStarted.md

```diff
@@ -1,55 +1,69 @@
 # Getting Started
 
-In this section, we will show you how to install and configure Gateway API via the Ingress Operator. We will configure
-a simple HTTPRoute and backend to demo basic Gateway API functionality.
+In this section, we will show you how to install and configure Gateway API via the Ingress Operator. We will configure a simple HTTPRoute and backend to demo basic Gateway API functionality.
 
-Note: Since this is still a Developer Preview, it is important to note that it may not be flawless, and it is possible
-for bugs to be present.
+Note: Since this is still a Developer Preview, it is important to note that that it may not be flawless, and it is possible for bugs to be present.
```

Comment:
Since we now have a GA, we'll need to reword alot of this.
```suggestion
Gateway API via Ingress Operator is GA in OpenShift 4.19.  These intstructions were written for versions prior 4.19, when Gateway API was considered a Developer Preview.  When using versions prior to 4.19, it is important to note that it is possible for bugs to be present.
```